src/cloudWatchLogs/commands/saveCurrentLogDataContent.ts

```diff
@@ -7,29 +7,25 @@ import * as vscode from 'vscode'
 import * as nls from 'vscode-nls'
 const localize = nls.loadMessageBundle()
 
-import * as fs from 'fs-extra'
 import { SystemUtilities } from '../../shared/systemUtilities'
 import { isLogStreamUri, parseCloudWatchLogsUri } from '../cloudWatchLogsUtils'
-import { LogDataRegistry } from '../registry/logDataRegistry'
 import { telemetry, CloudWatchResourceType, Result } from '../../shared/telemetry/telemetry'
-import { generateTextFromLogEvents } from '../document/textContent'
+import { FileSystemCommon } from '../../srcShared/fs'
 
-export async function saveCurrentLogDataContent(uri: vscode.Uri | undefined, registry: LogDataRegistry): Promise<void> {
+/** Saves the current "aws-cwl:" document. */
```

Comment:
Could we give more detail with something like:

```suggestion
/** Prompts the user to select a file location to save the currently visible "aws-cwl:" document to. */
```